experimental/packages/opentelemetry-sdk-metrics-base/src/export/ConsoleMetricExporter.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ExportResult, ExportResultCode } from '@opentelemetry/core';
+import { InstrumentType } from '../InstrumentDescriptor';
+import { AggregationTemporality } from './AggregationTemporality';
+import { ResourceMetrics, DataPointType } from './MetricData';
+import { PushMetricExporter } from './MetricExporter';
+
+/* eslint-disable no-console */
+export class ConsoleMetricExporter implements PushMetricExporter {
+  protected _shutdown = false;
+
+  export(metrics: ResourceMetrics, resultCallback: (result: ExportResult) => void): void {
+    return ConsoleMetricExporter._sendMetrics(metrics, resultCallback);
+  }
```

Comment:
Don't export if already shutdown

```suggestion
  export(metrics: ResourceMetrics, resultCallback: (result: ExportResult) => void): void {
    if (this._isShutdown) {
    	setImmediate(resultCallback, { code: ExportResultCode.SUCCESS });
    	return;
    }
    return ConsoleMetricExporter._sendMetrics(metrics, resultCallback);
  }
```